loris/resolver.py

```diff
@@ -309,14 +305,9 @@ def cached_files_for_ident(self, ident):
     def in_cache(self, ident):
         cache_dir = self.cache_dir_path(ident)
         if exists(cache_dir):
```

Comment:
This can just return `exists(self.cache_dir_path(ident))` rather than the `if/else`, right?